design.md

```diff
@@ -0,0 +1,8 @@
+Design which is in the below link is Admin related view.
+
+Design:- https://balsamiq.cloud/s165djm/p41w40c
+
+Flow diagram is the complete figure of the website.
+
+Flow diagram:- https://drive.google.com/file/d/1oNx0eAEpnyqtqz17votFGa7UD9SOK9nl/view?usp=sharing
```

Comment:
Do we really need this?

![image](https://user-images.githubusercontent.com/1830380/115238024-1e51e200-a115-11eb-84f3-e7c7857096ca.png)

Isn't it just directly dashboard?

Reply:
We can also have a decision here: If user is Admin, then both Admin & UserDashboard, else just UserDashboard.

![image](https://user-images.githubusercontent.com/1830380/115238147-42adbe80-a115-11eb-8f54-9f4a867e8d9c.png)

Also, where are the Onboarding Steps - that's the whole project about!

/cc @Insomaniacgamer

Reply:
> Do we really need this?
> 
> ![image](https://user-images.githubusercontent.com/1830380/115238024-1e51e200-a115-11eb-84f3-e7c7857096ca.png)
> 
> Isn't it just directly dashboard?

Do you mean by user profile @praveenscience

Reply:
Well, kind of! 🔥